tests/unit/s2n_tls13_record_aead_test.c

```diff
@@ -400,7 +400,7 @@ int main(int argc, char **argv)
             /* now if this was an application data, it cannot be parsed */
             EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->header_in));
             EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->in));
-            EXPECT_SUCCESS(s2n_stuffer_write_uint8(&conn->header_in, APPLICATION_DATA));
+            EXPECT_SUCCESS(s2n_stuffer_write_uint8(&conn->header_in, TLS_APPLICATION_DATA));
```

Comment:
Locally I can replace APPLICATION_DATA with any value and this test still passes (s2n_record_parse fails). It looks like the parsing fails at https://github.com/awslabs/s2n/blob/master/tls/s2n_record_read_aead.c#L86. I'm not sure this is testing what the comment indicates is being tested. I will follow up to make sure I'm following the code correctly.

Reply:
yes, the only time where the tests would fail is when you set it to the CCS type.